downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/FetchTestReactomeOntologyFiles.java

```diff
@@ -65,32 +65,32 @@ public static void execute(MySQLAdaptor dba, String releaseNumber) throws SQLExc
 				if (splitLine.length > 1 && splitLine[1].matches("( [A-Z][a-z]{2}){2} [0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2} [A-Z]{3} [0-9]{4}")) {
 					String dateTime = ";" + splitLine[1] + "\n";
 					dateTimeCounter++;
-					if (pprjSwitch && dateTimeCounter == 1) {
-						Files.write(Paths.get(pprjFilename), dateTime.getBytes(), StandardOpenOption.APPEND);
+					if (pinsSwitch && dateTimeCounter == 1) {
+						Files.write(Paths.get(pinsFilename), dateTime.getBytes(), StandardOpenOption.APPEND);
 					}
 					if (dateTimeCounter == 2) {
 						Files.write(Paths.get(pontFilename), dateTime.getBytes(), StandardOpenOption.APPEND);
 					}
 					if (dateTimeCounter == 3) {
-						Files.write(Paths.get(pinsFilename), dateTime.getBytes(), StandardOpenOption.APPEND);
+						Files.write(Paths.get(pprjFilename), dateTime.getBytes(), StandardOpenOption.APPEND);
 					}
 					continue;
 				}
 				line += "\n";
 
-				// Generate pprj file
-				if (dateTimeCounter == 1 && pprjSwitch) {
+				// Generate pins file
+				if (dateTimeCounter == 1 && pinsSwitch) { 
 					if (line.contains("pprj_file_content")) {
 						line = "\n";
-						pprjSwitch = false;
+						pinsSwitch = false;
 					}
 					if (line.contains(".pont")) {
 						line = line.replaceAll("[a-zA-Z0-9]+.pont", pontFilename);
 					}
 					if (line.contains(".pins")) {
-						line = line.replaceAll("[a-zA-Z0-9]+.pins", pinsFilename);
+						line = line.replaceAll("[a-zA-Z0-9]+.pins", pprjFilename);
```

Comment:
Why is *.pins replaced with the `pprjFilename` value (instead of the `pinsFilesname` value) but *.pont is replaced with the `pontFilename` value?

Reply:
I thought the pont file was created OK, just the pins and pprj had each others' names.

Reply:
Yeah, so I'm just confused why the pont file line is replaced with the pont value but the pins file line is replaced by the pprj value?  If they are getting inverted, maybe changing which variable gets assigned to what value is better instead of changing the value in the replacement?  It looks like it is mis-matched at the moment in this line, but I could be mis-understanding.

Reply:
You are right that this is confusing -- I think this is related to why this happened in the first place. On the OICR server, if you look at one of the perl-generated pins/pprj (release 66 or so) files, it showed the pprj and pins file contents opposite to how it was being produced by the java code. I think I originally designated the pins/pprj files based on parsing the object (so I assumed when the line contains pins that it was for the pins file). This is making me wonder if the perl way of doing it was actually populating the wrong files? I'll look into it

Reply:
OK so it turns out that that condition is never hit, so I have deleted it

Reply:
For further reading on the pprj, pins and pont files: https://protegewiki.stanford.edu/wiki/PrF_UG_files_protege_files